src/emoji/data.js

```diff
@@ -29,6 +29,32 @@ export const getFilteredEmojiNames = (
   query: string,
   activeImageEmojiByName: $ReadOnly<{ [string]: ImageEmojiType }>,
 ): string[] => {
+  // Check if the query itself is an emoji, and is known to Zulip. If
+  // so, return the emoji name in an array. Emoji with single-codepoint and
+  // multi-codepoint have length 2 and 3 respectively.
```

Comment:
"Emoji with single-codepoint and multi-codepoint have length 2 and 3 respectively."

Hmm, that's surprising to me! I wonder why? Maybe add a link to some documentation in this code comment?

Reply:
Even I am wrong. Emoji length can vary a lot depending on modifiers :sweat: 
https://hsivonen.fi/string-length/